### DISCUSSION TITLE
How can I do conditionals over the new Prisma v4 DbNull?

### DISCUSSION THREAD

**Discussion by PabloSzx** [prisma/prisma#14393] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/8672915/179867928-3c61100f-507b-46fc-b9cd-4783a3c8d29c.png)

**Comment by PabloSzx** [comment, marked as answer]
```ts
function isDbNull(value: unknown): value is typeof DbNull {
  return value === DbNull;
}

declare const foo: typeof DbNull | { hello: 123 };

if (isDbNull(foo)) {
} else {
  foo.hello;
}
```

ok, I found a workaround, but I would like to know if there is a better solution available instead of making a hand-made helper to work around this issue

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
